src/csmpagent/csmp_reportSubscribe.c

```diff
@@ -14,14 +14,14 @@
  *  limitations under the License.
  */
 
-#include <stdlib.h>
 #include <string.h>
 #include "csmp.h"
 #include "cgmsagent.h"
 #include "csmptlv.h"
 #include "csmpagent.h"
 #include "csmpfunction.h"
 #include "CsmpTlvs.pb-c.h"
+#include "osal_common.h"
```

Comment:
Need to update this appropriately after concluding osal file/dir structure mentioned in the previous comment.